src/conv/k-g/sphere.cpp

```diff
@@ -0,0 +1,100 @@
+/*                 S P H E R E . C P P
+ * BRL-CAD
+ *
+ * Copyright (c) 2024-2025 United States Government as represented by
+ * the U.S. Army Research Laboratory.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License
+ * version 2.1 as published by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this file; see the file named COPYING for more
+ * information.
+ */
+ /** @file sphere.cpp
+  *
+  * LS Dyna keyword file to BRL-CAD converter:
+  * intermediate sphere implementation
+  */
+
+#include "sphere.h"
+
+Sphere::Sphere(void) : m_name() {}
+
+void Sphere::setName
+(
+    const char* value
+) {
+    if (value != nullptr)
+	m_name = value;
+    else
+	m_name = "";
```

Comment:
I didn't check if the other entities do this but there is at least conceptually a difference between a null name and an empty string name.  If this is exclusively used for object-creation on the wdb side of things, then write() should probably auto-generate some unique string name (e.g., "unnamed_001_" monotonically increasing) so that it's uniquely identifiable in the .g database.  Otherwise the object will get overwritten with other ""-named sphere objects.